packages/corvid-cli/src/apps/open-editor.js

```diff
@@ -61,6 +67,10 @@ const openEditorApp = ({ useSsl = true } = {}) => ({
                 getMessage("OpenEditor_Client_Console_Fatal_Error")
               )
             );
+          },
+          [clientMessages.DECODE_ERROR_MESSAGE]: path => {
+            const message = `decode error in ${path}`;
```

Comment:
@wixysam and here

Reply:
Same questions as above.

Reply:
user actions or maybe old version of site and we had changed encoding function (shouldn't happen)

Reply:
Same idea as above: Error decoding ${path}. The file has been corrupted. Try reverting to an older version.

Reply:
there is a problem here, path is not a file and we don't even know to which file it will be converted :(. it is something like "pages.d1dmp.content"

Reply:
Then leave out the middle sentence. I guess that's the best we can do.

Reply:
Thanks!